cluster-scope/overlays/prod/moc/smaug/kustomization.yaml

```diff
@@ -24,7 +24,6 @@ resources:
 - ../../../../base/core/namespaces/aicoe-meteor
 - ../../../../base/core/namespaces/apicurio-apicurio-registry
 - ../../../../base/core/namespaces/b4mad-racing
-- ../../../../base/core/namespaces/cosigned
```

Comment:
should be renamed instead of being removed

Reply:
this is currently not needed in smaug cluster. 
removed for now, will add it when required.

Reply:
oops looks like this needs to be removed from the smaug cluster.